src/main/java/com/knighttodo/knighttodo/domain/DayTodoVO.java

```diff
@@ -0,0 +1,27 @@
+package com.knighttodo.knighttodo.domain;
+
+import com.knighttodo.knighttodo.gateway.privatedb.representation.enums.Hardness;
+import com.knighttodo.knighttodo.gateway.privatedb.representation.enums.Scariness;
+import lombok.*;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString
+public class DayTodoVO implements TodoVO {
+
+    private String id;
+
+    private String dayTodoName;
+
+    private Scariness scariness;
+
+    private Hardness hardness;
+
+    private boolean ready;
+
+    private DayVO day;
+
+    private Integer experience;
```

Comment:
Consider to use primitive type if experience cannot be null

Reply:
It's supposed to be used as an Integer but I'll use Optional to prevent NPE